tools/scripts/e2e_test.sh

```
@@ -6,6 +6,83 @@
+        .items
+      elif type == "object" and (.workers | type) == "array" then
+        .workers
+      else
+        []
+      end;
+    [worker_items[] | select(.pool == $pool)] | length > 0
+  ' >/dev/null
+}
+
+if [[ "${CORDUM_E2E_SOURCE_ONLY:-}" == "1" ]]; then
+  return 0 2>/dev/null || exit 0
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)